test-utils/runtime-tester/src/run_test.rs

```diff
@@ -119,6 +122,17 @@ impl TransactionConfig {
     }
 }
 
+use core::fmt::Debug;
+impl Debug for TransactionConfig {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        write!(
+            f,
+            "(Nonce: {}, SignerId: {}, ReceiverId: {}, Actions: {:?} )",
+            &self.nonce, &self.signer_id, &self.receiver_id, &self.actions
+        )
+    }
+}
```

Comment:
It's better to `use std::fmt` once. 

```suggestion
impl fmt::Debug for TransactionConfig {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        write!(
            f,
            "(Nonce: {}, SignerId: {}, ReceiverId: {}, Actions: {:?} )",
            &self.nonce, &self.signer_id, &self.receiver_id, &self.actions
        )
    }
}
```

Reply:
I changed the way Scenario Debug works, but used std::fmt this time